internal/experiment/webconnectivityqa/httpdiff.go

```diff
@@ -83,18 +83,18 @@ func httpDiffWithInconsistentDNS() *TestCase {
 		ExpectErr: false,
 		ExpectTestKeys: &testKeys{
 			DNSExperimentFailure:  nil,
-			DNSConsistency:        "inconsistent",
+			DNSConsistency:        "consistent",
```

Comment:
Here the DNS flips with consistent because the proxy we're redirecting to supports TLS.

Reply:
**TODO** split this test case into two test cases